Changelog.md

```diff
@@ -15,6 +15,7 @@
 - Ensure bonus marks are not included in assignment "out of" in submissions table (#6836)
 - Ensure assignment "out of" in submissions table is rounded to two decimal places (#6836)
 - Added POST API for Group Creation (#6834)
+- Allow inactive groups in the graders table to be toggled for display (#6778)
```

Comment:
We've updated the changelog with a new minor release. Please do a merge from master; after that, make sure to move this entry so it appears under the "Unreleased" section.